synapsebridgehelpers/summaryTable.py

```diff
@@ -0,0 +1,33 @@
+import synapsebridgehelpers
+import synapseclient
+import pandas as pd
+
+def summaryTable(syn, projectId, columns = []):
+    """Outputs a concatenated table containing the given
+    list of columns from the given projectId. If no columns 
+    are given, then all columns are considered
+    
+    NOTE: When giving a column, note that it needs to be 
+    present in all the tables of the given Project, otherwise
+    the function will throw an error
+    
+    Arguments:
+    - syn: a Synapse client object
+    - projectID: synapse ID of the project we want to summarize
+    - columns: list of columns we want in the summary table"""
+    
+    all_tables = synapsebridgehelpers.get_tables(syn, projectId)
+    df_list = []
+    columns_str = ','.join(columns)
+    # Change the line below to edit default columns
+    columns_str_default = 'recordId,healthCode,externalId,uploadDate'
+    columns_str = columns_str_default if columns_str == '' else columns_str # If empty then we choose default columns
```

Comment:
We should determine which are intersecting with the asked columns and query on those:

syn.getColumns(...)